apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -626,3 +636,71 @@ function wpcom_site_has_global_styles_feature( $blog_id = 0 ) {
 
 	return false;
 }
+
+/**
+ * Checks whether the site has access to the Global Styles feature when the editor is live previewing a Woo theme without a Business plan or higher.
+ *
+ * @param int $blog_id The WPCOM blog ID.
+ * @return bool Whether the site has access to Global Styles when live previewing.
+ */
+function wpcom_global_styles_is_previewing_woo_theme_without_business_plan( $blog_id ) {
```

Comment:
FYI, this might be changed in the future so the user can preview bundle themes instead of woo themes.  CC: @Automattic/lego 
- paYJgx-4mO-p2
- https://github.com/Automattic/wp-calypso/pull/84483